clippy.toml

```diff
@@ -0,0 +1,9 @@
+disallowed-methods = [
+    # `ON CONFLICT ... WHERE ... DO UPDATE` does not work with CockroachDB --
+    # see https://github.com/oxidecomputer/omicron/issues/5047.
+    #
+    # Instead, import diesel::query_dsl::methods::FilterDsl, then use
+    # `do_update().set(...).filter(...)`. (This is not part of the
+    # documentation as of Diesel 2.1.4, but it works.)
+    "diesel::upsert::DecoratableTarget::filter_target",
```

Comment:
I wonder if it's possible to write a simple test which would break if, when updating to a new Diesel version, `FilterDsl`  changed to no longer generate SQL that works correctly with crdb? Is that worth the effort?

Reply:
I think existing tests would catch this, right?